ext/ddtrace_profiling_native_extension/private_vm_api_access.c

```diff
@@ -19,6 +19,8 @@
   #pragma GCC diagnostic push
   #pragma GCC diagnostic ignored "-Wunused-parameter"
   #pragma GCC diagnostic ignored "-Wattributes"
+  #pragma GCC diagnostic ignored "-Wpragmas"
+  #pragma GCC diagnostic ignored "-Wexpansion-to-defined"
```

Comment:
Why are these being ignored?
Would it make sense to write a short comment about each of these `GCC diagnostic ignored`?